Backend/app/routes/collaboration.py

```
@@ -0,0 +1,18 @@
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy import update
+from ..db.db import get_db
+from ..models.models import Collaboration # Verify this name in models/models.py
+
+router = APIRouter(prefix="/collaboration", tags=["Collaboration"])
+
+@router.put("/update-status/{collab_id}")
+async def update_collab_status(collab_id: str, status: str, db: AsyncSession = Depends(get_db)):
```

Ruff (0.15.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)